Misc/NEWS.d/next/Tests/2021-09-25-22-22-40.bpo-41914.O-QsyY.rst

```diff
@@ -0,0 +1,4 @@
+Fix issue running test_pdb inside GNU `screen`: for some reason, two tests
```

Comment:
I'd remove everything after the first colon, NEWS entries just need to say what the change is, not a detailed motivation.